frontend/src/scenes/activity/live/LiveEventsTable.tsx

```diff
@@ -17,7 +20,27 @@ const columns: LemonTableColumns<LiveEvent> = [
         key: 'event',
         className: 'max-w-80',
         render: function Render(_, event: LiveEvent) {
-            return <PropertyKeyInfo value={event.event} type={TaxonomicFilterGroupType.Events} />
+            const searchUrl = urls.activity(ActivityTab.ExploreEvents) + `?q=${encodeURIComponent(JSON.stringify({
+                kind: 'DataTableNode',
+                full: true,
+                source: {
+                    kind: 'EventsQuery',
+                    properties: [
+                        {
+                            key: '$event_uuid',
+                            operator: PropertyOperator.Exact,
+                            type: 'event',
+                            value: event.uuid
+                        }
+                    ]
+                },
+                propertiesViaUrl: true
+            }))}`
+            return (
+                <Link to={searchUrl}>
```

Comment:
We should instead implement the "Copy link to event" pattern, because this inline link fundamentally doesn't work: at the point in time the user sees a live event come in, that event does not yet exist in ClickHouse for a minute or so (depending on ingestion lag), so this link is effectively a 404

Reply:
Same as in DataTable: [code](https://github.com/PostHog/posthog/blob/5a6b7759b2f2bcecda702d7bdc3ce1b0713fb286/frontend/src/queries/nodes/DataTable/EventRowActions.tsx#L44-L56)

Reply:
right, that's why i made it a search instead of direct to the link, so you can search and refresh until it comes in - maybe we just need a user friendly way of telling users that it might be delayed though

copy link to event has the same issue right?

Reply:
ok, in the end i went with your way 1) for consistency, and 2) because we already have a tooltip on the event name and adding a second tooltip didn't work - and i wanted the warning tooltip saying it might not show up

see screenshot!